### ISSUE TITLE
Title formatting 2.0 in shellexec

### ISSUE THREAD

**Issue by mallniya** [Alexey-Yakovenko/deadbeef#1556]
As I understand, shellexec for whatever reason doesn't use title formating 2.0, for example instead of using command `nautilus %path%`, one must use `nautilus %F`. Please make shellexec plugin coherent to common title formating.

**Comment by Alexey-Yakovenko** [comment]
Yeah, shellexec is much harder to convert, so this is on 0.8 roadmap, but can't be done in 0.7 series without braking too much stuff.
